package.json

```diff
@@ -69,6 +69,7 @@
     "moment": "^2.29.1",
     "next": "^12.1.0",
     "next-images": "^1.8.1",
+    "nextjs-current-url": "^1.0.3",
```

Comment:
@ratishjain12 did you use this lib in the project?

Reply:
Yeah I will remove this its not needed